src/sections/Partners/partners.js

```diff
@@ -151,7 +152,18 @@ export const partners = [
     //imageRoute: "https://www.vmware.com/",
     moreInfo:
       "Layer5 enhances the Tanzu Service Mesh experience with Meshery as an application and infrastructure management platform that accelerates the adoption of open-source and cloud-native technologies by eliminating complexity with its intuitive, efficient, and flexible solution. Using Layer5's solutions along with Tanzu, engineers can self-service while enterprise IT teams offer Kubernetes as a Service to their application service teams.",
+  }, {
+    name: "Metabit Trading",
+    desc: "Metabit Trading is a technology-driven quantitative investment firm harnessing the power of artificial intelligence and machine learning.",
```

Comment:
Thanks for rustling up a description, @arunkarthicknm 👏

Reply:
Ahh yes! 
Pardon me, will do it rn 😄 @Udit-takkar

Reply:
> Thanks for rustling up a description, @arunkarthicknm 👏

My pleasure! @leecalcote :)